buildpack.md

```diff
@@ -542,11 +551,19 @@ For each entry in `<plan>`:
 
 #### Bills-of-Materials
 
-When the build is complete, a BOM (Bill-of-Materials) describing the app image MAY be generated for auditing purposes.
-If generated, this BOM MUST contain all `bom` entries in each `launch.toml` at the end of each `/bin/build` execution, in adherence with the process and data format outlined in the [Platform Interface Specification](platform.md).
+Buildpacks MAY write standardized Bill of Materials (sBOM) files with extension `<ext>`, where `<ext>` MUST denote an sBOM media type based on Internet Assigned Numbers Authority (IANA) [assigned media types](https://www.iana.org/assignments/media-types/media-types.xhtml). The currently supported media types and their expected file extensions are as follows:
 
-When the build is complete, a build BOM describing the build container MAY be generated for auditing purposes.
-If generated, this build BOM MUST contain all `bom` entries in each `build.toml` at the end of each `/bin/build` execution, in adherence with the process and data format outlined in the [Platform Interface Specification](platform.md).
+ | sBOM Media Type                  | File Extension
+ |----------------------------------|----------------------------------------------
+ | `application/vnd.cyclonedx+json` | `cdx.json`
+ | `application/spdx+json`          | `spdx.json`
+ | `application/vnd.syft+json`      | `syft.json`
+
+When the build is complete, an sBOM describing the app image MAY be generated for auditing purposes.
+If generated, this BOM MUST contain all `<layer>.bom.<ext>` files for each `launch = true` layer at the end of each `/bin/build` execution, as well as `launch.bom.<ext>` if provided, in adherence with the process and data format outlined in the [Platform Interface Specification](platform.md).
+
+When the build is complete, a build sBOM describing the build container MAY be generated for auditing purposes.
```

Comment:
This sentence and that in line 562 appear to be duplicates until you realize that one is talking about launch vs build. Maybe we can emphasize the difference through style:

```suggestion
When the build is complete, a **build sBOM** describing the build container MAY be generated for auditing purposes.
```